packages/components/stamp/src/stamp.story.js

```diff
@@ -32,11 +32,11 @@ storiesOf('Components|Stamps', module)
               <Stamp tone={tone} isCondensed={boolean('isCondensed', false)}>
                 <SpacingsInline alignItems="center">
                   <Icon size="medium" />
```

Comment:
Perhaps we can also use the `tone` for the icons so they look in the same color as the text.

Reply:
Thank you for the feedback.
All updated here: https://github.com/commercetools/ui-kit/pull/2370/commits/3645eea46de1187e8e0195d392fe8f7e50b83bde